src/devices/LD_MRS.cpp

```diff
@@ -450,6 +450,25 @@ std::string LDMRS::getFirmwareVersion()
 	}
 }
 
+std::string LDMRS::getFPGAVersion()
+{
+	if (m_lux == NULL)
+	{
+		return "(none)";
+	} else {
+		return m_lux->getFPGAVersion();
+	}
+}
+
+std::string LDMRS::getTemperature()
```

Comment:
why not return the double?

Reply:
It's just for the diagnosis. But a double is as good or even better. Please do not hesitate to change the return value. Btw, thanks for the excellent work in libsick_ldmrs. It's a pleasure to use such a well structured project!

Reply:
it's your PR, so please go ahead ;).

Reply:
Done :)